src/ProcessManager.php

```diff
@@ -675,6 +623,8 @@ protected function commandRegister(array $data, ConnectionInterface $conn)
             $slave = $this->slaves->getByPort($port);
             $slave->register($pid, $conn);
         } catch (\Exception $e) {
+            $this->output->writeln($e->getMessage());
```

Comment:
Why are we showing this if there's only one reason for the exception as given below?